src/data/features/mondstadt/boss/andrius.json

```diff
@@ -1,5 +1,8 @@
 {
-  "name": { "en": "Andrius (Lupus Boreas)" },
+  "name": { 
+    "en": "Andrius (Lupus Boreas)",
+    "ru": "Андриус (Лупус Бореалис)",
```

Comment:
The comma at the end of this line needs to be removed.

Reply:
Don't worry about it, not hard to help with this ;) And thank you for remark, i didn't know about this  JSON syntax nuance.